Solutions/VMware SD-WAN and SASE/SolutionMetadata.json

```diff
@@ -0,0 +1,16 @@
+{
+	"publisherId": "velocloud",
+	"offerId": "azure-sentinel-solution-vmwaresase",
+	"firstPublishDate": "2023-12-31",
+	"providers": ["VMware"],
+	"categories": {
+		"domains" : ["Security - Threat Protection"],
+		"verticals": []
+	},
+	"support": {
```

Comment:
Just wanted to confirm Microsoft team provide support to this solution

Reply:
It should be partner supported.